internal/common/docker/images_test.go

```diff
@@ -152,6 +170,7 @@ func TestParseImageName(t *testing.T) {
 			image, err := ParseImageName(tt.image)
 			if !tt.wantErr {
 				assert.NoError(t, err, "ParseImageName() error = %v, wantErr %v", err, tt.wantErr)
+				assert.Equal(t, tt.wantImage, image.Repository, "ParseImageName() image reference = %v, want %v", image.Repository, tt.wantImage)
```

Comment:
In all test cases `wantRepository` is the same as `wantImage` and is subsequently compared to the same field. Additionally, `Image` is not a separate field of [`ImageRef`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/e0646e1c227019fc90576f566ef005b44e572f7d/internal/common/docker/images.go#L13) from what I can see. 

What's the reason for a separate field? Maybe we'd be better off changing the test case struct to have `wantImageRef` and specify a whole `ImageRef` struct.